src/Sylius/Bundle/CoreBundle/Settings/GeneralSettingsSchema.php

```diff
@@ -67,38 +75,64 @@ public function buildSettings(SettingsBuilderInterface $builder)
     {
         $builder
-            ->add('title', 'text', array(
-                'label'       => 'sylius.form.settings.general.title',
-                'constraints' => array(
-                    new NotBlank()
+            ->add(
+                'title',
+                'text',
+                array(
+                    'label' => 'sylius.form.settings.general.title',
+                    'constraints' => array(
+                        new NotBlank()
+                    )
+                )
+            )
+            ->add(
+                'meta_keywords',
+                'text',
+                array(
+                    'label' => 'sylius.form.settings.general.meta_keywords',
+                    'constraints' => array(
+                        new NotBlank()
+                    )
                 )
-            ))
-            ->add('meta_keywords', 'text', array(
-                'label'       => 'sylius.form.settings.general.meta_keywords',
-                'constraints' => array(
-                    new NotBlank()
+            )
+            ->add(
+                'meta_description',
+                'textarea',
+                array(
+                    'label' => 'sylius.form.settings.general.meta_description',
+                    'constraints' => array(
+                        new NotBlank()
+                    )
                 )
-            ))
-            ->add('meta_description', 'textarea', array(
-                'label'       => 'sylius.form.settings.general.meta_description',
-                'constraints' => array(
-                    new NotBlank()
+            )
+            ->add(
+                'locale',
+                'locale',
+                array(
+                    'label' => 'sylius.form.settings.general.locale',
+                    'constraints' => array(
+                        new NotBlank(),
+                        new Locale(),
+                    )
                 )
-            ))
-            ->add('locale', 'locale', array(
-                'label'       => 'sylius.form.settings.general.locale',
-                'constraints' => array(
-                    new NotBlank(),
-                    new Locale(),
+            )
+            ->add(
+                'currency',
+                'sylius_currency_choice',
+                array(
+                    'label' => 'sylius.form.settings.general.currency',
+                    'constraints' => array(
+                        new NotBlank(),
+                        new Currency(),
+                    )
                 )
-            ))
-            ->add('currency', 'sylius_currency_choice', array(
-                'label'       => 'sylius.form.settings.general.currency',
-                'constraints' => array(
-                    new NotBlank(),
-                    new Currency(),
+            )
+            ->add(
+                'google_analytics_code',
+                'text',
```

Comment:
The text form type but you seem to use it as boolean in your template. do I miss something ?